src/components/Checkbox/Checkbox.story.js

```diff
@@ -15,16 +16,16 @@ storiesOf('Checkbox', module)
       <State
         initial={false}
         name="isChecked"
-        updaterName="onToggle"
+        updaterName="onChange"
         updater={isChecked => !isChecked}
       >
-        {({ isChecked, onToggle }) => (
+        {({ isChecked, onChange }) => (
           <Checkbox
-            value="checkbox"
+            value={isChecked}
```

Comment:
Mhm. This is still not how I would expect the value to be used. 🤔 According to the [specs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#Value), this is meant to be an identifier for the input, not the input state.

Reply:
Please see above :) I believe this is compatible with the two defined behaviors of final form's checkboxes.